src/Kunstmaan/MediaBundle/Helper/File/FileHandler.php

```diff
@@ -59,15 +60,23 @@ class FileHandler extends AbstractMediaHandler
     /**
      * Constructor
      *
-     * @param int                              $priority
-     * @param MimeTypeGuesserFactoryInterface  $mimeTypeGuesserFactory
-     * @param ExtensionGuesserFactoryInterface $extensionGuesserFactoryInterface
+     * @param int                                   $priority
+     * @param MimeTypeGuesserFactoryInterface       $mimeTypeGuesserFactory
+     * @param ExtensionGuesserFactoryInterface|null $extensionGuesserFactoryInterface
      */
-    public function __construct($priority, MimeTypeGuesserFactoryInterface $mimeTypeGuesserFactory, ExtensionGuesserFactoryInterface $extensionGuesserFactoryInterface)
```

Comment:
We have rethink this change as changing the constructor parameters does also change the public properties and as they're public it's a bc break 🙁 (same for the ImageHandler extending from this class) 

I will also try to come up with a non bc-break solution here. I would also avoid injecting the `@mime_types` service twice, we only need it once